lib/elixir/lib/io/ansi/docs.ex

```diff
@@ -291,6 +291,8 @@ defmodule IO.ANSI.Docs do
     col =
       col
       |> String.replace("\\\|", "|")
+      |> String.replace(~r/-+/, "-")
```

Comment:
@josevalim We can probably do it by excluding the header lines from the length count. That gets rid of the regex and the need to trip down the "-".